generated/nidaqmx/nidaqmx.proto

```diff
@@ -238,8 +242,22 @@ service NiDAQmx {
   rpc WriteToTEDSFromFile(WriteToTEDSFromFileRequest) returns (WriteToTEDSFromFileResponse);
 }
 
-enum NiDAQmxAttributes {
-  NIDAQMX_ATTRIBUTE_UNSPECIFIED = 0;
+enum ScaleAttributes {
```

Comment:
I realized from this work that I misunderstood how attributes work. And, particularly, the differences from enums.

I am planning to add the following features (only for DAQ, for now):
1. Generate a separate attributes enum for each datatype: ScaleInt32Attributes, ScaleDoubleAttributes, etc.
   * This gives better auto-complete. This is especially important because DAQ will not report errors if you use the wrong datatype.
 2. Have an `attribute_raw`/oneof for attributes similar to enums.
    * We need this for internal attributes and attributes-we-don't-know-about.

Any concerns about those ideas?

Reply:
This would be in a future PR.

Reply:
Those ideas sound good to me, especially for DAQ because of the reasons you laid out.

Reply:
Good point @doshirohan , I think you're right.